### PR TITLE
[Chore] Cleanup unnecessary code and files

### PR DESCRIPTION
**Pull request by oreoluwade**
#### What does this PR do?
- Cleans out unnecessary packages and files 

#### Description of Task to be completed?
- Deprecate redux 
- Remove unnecessary packages, files and directories.

#### How should this be manually tested?
- Run the app as normal, test the app as normal

#### What are the relevant pivotal tracker stories?
- N/A

### CODE REVIEW

File: src/components/Authentication/Login/login-form.jsx [hunk 1]
```diff
@@ -1,143 +1,153 @@
...
+import { Divider, Button, Typography } from "@material-ui/core";
+import { func, object, loginFormValidator } from "src/util";
+import { Loader, InputBox } from "components/base";
+import { LOGIN_MUTATION } from "src/queries";
+import { encrypt } from "src/util";
 
 const INITIAL_VALUES = {
-  identifier: '',
-  password: ''
+  identifier: "",
```
Comment (fisayomi) [inline src/components/Authentication/Login/login-form.jsx]: Are we switching to double quotes?

File: src/components/protected/protected.jsx [hunk 2]
```diff
@@ -1,23 +1,46 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React, { useReducer, useEffect } from 'react';
```
Comment (omorilewa) [inline src/components/protected/protected.jsx]: I

File: src/components/Authentication/Login/login-form.jsx (hunk 1 above)
Comment (oreoluwade) [reply src/components/Authentication/Login/login-form.jsx]: Hmmm. Good question. I don't have a preference, tbh. My last two PRs have been double quotes, and I think this is because prettier formats the docs to use double quotes by default. I could change it if "we" prefer single quotes though. What do you think @omorilewa

Comment (fisayomi) [reply src/components/Authentication/Login/login-form.jsx]: I think I prefer single quotes, eslint is configured to use single quote

File: src/components/protected/protected.jsx (hunk 2 above)
Comment (oreoluwade) [reply src/components/protected/protected.jsx]: So I'd love for there to be either a `user provider` (either with an HOC or a user context provider) that works separately from the ProtectedRoute. And the reason this is important is so that once a user is authenticated, all protected pages should have access to the user object to prevent doing user queries on every page.

TLDR, the Protected can do the job it was doing before; simply render if the user is authenticated. But if that's the Protected's only job, we'd have to create something that queries the details once and for all and passes it down as a prop to all components that need protection.
